src/containers/SignUp/Form/Form.tsx

```diff
@@ -19,20 +25,59 @@ export default function Form() {
     phoneNumber: "",
     grade: 0,
     gender: "",
+    profilePath: "조만간사라질예정",
+  });
+
+  const [passwordType, setPasswordType] = useState("password");
```

Comment:
```suggestion
  const [passwordType, setPasswordType] = useState<"password" | "text">("password");
```
이렇게 해놔야 혹시 다른 사람이 작업할때 혼동이 없을듯